farmdata2_modules/fd2_tabs/fd2_barn_kit/seedingReport.html

```diff
@@ -494,7 +495,23 @@ <h1 class="text-center">Seeding Report</h1>
                     let tableData = this.tableRows
                     let sumRowFeet = 0
                     tableData.map(h => {
-                        sumRowFeet = sumRowFeet + parseInt(h.data[4])
+                        sumRowFeet = sumRowFeet + parseFloat(h.data[4])
+                    })
+                    return (Math.round(sumRowFeet*100))/100
+                },
+                /**
+                 * returns the total row feet for Direct Seeding
+                 */ 
+                totalRowFeetNoLabor(){
+                    let tableData = this.tableRows
+                    let sumRowFeet = 0
+                    tableData.map(h => {
+                        if(parseFloat(h.data[10]) == 0 && parseFloat(h.data[11]) == 0) {
```

Comment:
You can invert the if condition here and just use an if instead of an if/else.  The if part will then add to sumRowFeed and the statement in the if: sumRowFeet = sumRowFeet that does not do anything will be removed.